mysql-replication/connector-config-master.json

```diff
@@ -3,8 +3,8 @@
   "tasks.max": "1",
   "database.hostname": "mysql-master",
   "database.port": "3306",
-  "database.user": "debezium",
-  "database.password": "dbz",
+  "database.user": "mysqluser",
+  "database.password": "mysqlpw",
```

Comment:
You should actually use Debezium user as it has [replication rights](https://github.com/debezium/container-images/blob/main/examples/mysql-replication/master/3.1/inventory.sql#L16)